lib/sovren-rest/client.rb

```diff
@@ -22,12 +22,14 @@ class Client
     #     account_id    #  Sovren account ID. If self hosted, not required
     #     service_key   #  Sovren service key. If self hosted, not required
     #     configuration #  Sovren parser configuration string. Optional
+    #     addl_headers  #  Additional headers for the remote call. Optional
     #   }
     def initialize(options)
       @base_url = options[:base_url]
       @account_id = options[:account_id] || '12345678'
       @service_key = options[:service_key] || 'thiscanbeanything,whyrequireit?'
       @configuration = options[:configuration] || ''
+      @addl_headers = options[:addl_headers] || {}
```

Comment:
I think the name here could be better. `parse_request_headers`?

Reply:
you think the naming needs to suggest they are in addition to the default headers already specified by the gem? additional_parse_request_headers is getting pretty long though haha